README.md

```diff
@@ -1,25 +1,25 @@
-# uds-core monorepo
+# Unicorn Delivery Service - Core (UDS Core)
 
 > [!WARNING]  
-> uds-core is in early alpha and is not ready for general consumption.
+> UDS Core is in early alpha and is not ready for general use.
 
-The UDS Core Bundle groups foundational Unicorn Delivery Service applications that are heavily influence by PlatformOne's [Big Bang](https://repo1.dso.mil/big-bang/bigbang).
+UDS Core groups foundational Unicorn Delivery Service applications that are heavily influenced [Big Bang](https://repo1.dso.mil/big-bang/bigbang).
 
 The core applications are:
 
-- [ ] Authservice
-- [x] Grafana
-- [x] Istio
-- [ ] KeyCloak
-- [ ] Kiali
-- [ ] Kyverno
-- [x] Loki
-- [x] Metrics Server
-- [x] Neuvector
-- [x] Prometheus
-- [x] Promtail
-- [ ] Tempo
-- [ ] Velero
+- [Authservice](https://github.com/istio-ecosystem/authservice) - Authorization
+- [Grafana](https://grafana.com/oss/grafana/) - Monitoring
+- [Istio](https://istio.io/) - Service Mesh
+- [KeyCloak](https://www.keycloak.org/) - Identity & Access Management
+- [Kiali](https://kiali.io/) - Service Mesh Observability
+- [Kyverno](https://kyverno.io/) - Policy Engine
+- [Loki](https://grafana.com/oss/loki/) - Log Aggregation
+- [Metrics Server](https://github.com/kubernetes-sigs/metrics-server) - Metrics
+- [Neuvector](https://open-docs.neuvector.com/) - Container Security
+- [Prometheus Stack](https://github.com/prometheus-operator/kube-prometheus) - Monitoring
+- [Promtail](https://grafana.com/docs/loki/latest/send-data/promtail/) - Log Aggregation
+- [Tempo](https://grafana.com/docs/tempo/latest/getting-started/) - Tracing
+- [Velero](https://velero.io/) - Backup & Restore
```

Comment:
Is there still value in capturing status here (the checkboxes)?

Reply:
maybe? idk

Reply:
Resolving - checkboxes would be pretty shortlived likely since we are going to knock the others out soon hopefully.